example/models/transaction_db.go

```diff
@@ -0,0 +1,27 @@
+package models
+
+import (
+	"github.com/arcana-network/groot/example/database"
+	"github.com/arcana-network/groot/logger"
+)
+
+type DBTransaction struct {
+	log logger.Logger
+	db  database.DB
+}
+
+func NewDBTransaction(db database.DB, log logger.Logger) TransactionModel {
+	return DBTransaction{
+		log: log,
+		db:  db,
+	}
+}
+
+func (t *DBTransaction) getTransactionAmount(userName string) int {
+	// Logic to retrieve txns from DB and assemble.
+	t.log.Info("Getting transaction details", logger.Field{
+		"username": userName,
+	})
+
+	return 20
+}
```

Comment:
Don't see this being used anywhere. Is there a reason to add this in the example

Reply:
The example is just to show how could we pass around logger in a complex project. It's not meant to work, it's just stubs as mentioned in PR description.
Nevertheless I think the reason to add it is just return some int as per function signature.

Reply:
The intention of example is only to show how to initialize/pass around logger. If it looks dirty, I can clean it up later.

Reply:
Added to backlog incase we need to do it, resolving this and merging for further PRs.

https://team-1624093970686.atlassian.net/browse/AR-1909?atlOrigin=eyJpIjoiY2FhZGQwN2RhMzIwNDJlNDlkYmNiYWM4ZGM0ZTc0MDAiLCJwIjoiaiJ9